### ISSUE TITLE
Get application hosted on Heroku

### ISSUE THREAD

**Issue by hello-consumer** [CodingTemple/Coding_Temple_Staff_Portal#1]
- [x] Make sure app builds and all dependencies are accounted for.
- [x] Push to Heroku
- [x] Move and scale up to allow for group access

**Comment by hello-consumer** [comment]
App is presently deployed at https://obscure-scrubland-97266.herokuapp.com/

I've got it hooked up to a Heroku hosted Postgresql instance, and it seems pretty stable.
Current issues I'm working through:
- [ ] Gunicorn "forks" 3-4 processes per application, and cookie/session information does not work correctly across forked processes (see: https://devcenter.heroku.com/articles/python-gunicorn) 
I've got concurrency set to 1 for the moment while I figure out what middleware I need to configure to get it working across forks.

- [ ] The "AdminForm" includes a dynamic drop-down list which breaks the initial migration, so that line needs to be commented/uncommented when first setting up the database on Postgres.  (It works fine for SQLLite).
